include/crocoddyl/core/numdiff/diff-action.hxx

```diff
@@ -99,22 +141,55 @@ void DifferentialActionModelNumDiffTpl<Scalar>::calcDiff(const boost::shared_ptr
     model_->calc(d->data_u[iu], x, u + d->du);
 
     const VectorXs& xn = d->data_u[iu]->xout;
-    const Scalar c = d->data_u[iu]->cost;
+    const Scalar cp = d->data_u[iu]->cost;
     data->Fu.col(iu) = (xn - xn0) / disturbance_;
 
-    data->Lu(iu) = (c - c0) / disturbance_;
+    data->Lu(iu) = (cp - c0) / disturbance_;
     d->Ru.col(iu) = (d->data_u[iu]->r - d->data_0->r) / disturbance_;
+
+    // We can apply the same formulas for finite difference as above
+
```

Comment:
Add the comment as in the action case. Same for the doxygen code inside the hpp file.